### ISSUE TITLE
NTR, pulmo: Abnormality of alveolar type II pneumocytes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6388]
**Preferred term label:** Abnormality of alveolar type II pneumocytes

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary alveolar system morphology ?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
IUs this the same as "Hyperplasia of type II alveolar cells" in https://pubmed.ncbi.nlm.nih.gov/19100526/ ?

**Comment by GrieseAG** [comment]
No it should act as parent term of 
- Hyperplasia of type II alveolar cells
- Hypertrophic alveolar type II pneumocytes

**Comment by mgriese** [comment]
(Pulmonary/alveolar/alveolar epithelial) Type II cells 
are
synonyms for (alveolar/alveolar epithelial) Type II pneumocytes.

**Comment by pnrobinson** [comment]
@GrieseAG 
extending the definition to provide more context for non-specialists, is this OK?

def: Any structural abnormality of alveolar type 2 (ATII) cells. 

Comment: Alveolar Type 1 (ATI) ATII cells are specialized epithelial cells of the distal lung. ATI cells are flattened squamous cells that cover around 95% of the alveolar surface and lie adjacent to capillary endothelial cells to form the pulmonary gas exchange region. ATII cells have a compact morphology and cover the remaining 5% of the alveolar surface. The ultrastructural hallmark of ATII cells is the expression of multilamellar bodies (MLB) containing dipalmitoylphosphatidyl choline (DPCC), the major lipid component of pulmonary surfactant that reduces surface tension in the alveoli to prevent collapse of the lungs at the end of expiration. ATII cells play an important role in innate immune responses within the lung with evidence that lung surfactant proteins have anti-microbial effects and reduce inflammation caused by the inhalation of irritants.  PMID:27792742

Adding synonyms according to your suggestions above!

Also, do we need to have anything for AT1 cells?

**Comment by mgriese** [comment]
OK, however  "multilamellar" bodies (MLB) is not used, instead  lamellar bodies (LB)
As general reference to the surfactant system you may include:
PMID: 10445627

There is an important morphological feature of abnormal LB´s: 
in patients with ABCA3 deficiency these appear as 
electron-dense bodies with central or peripheral core structures

PubMed: 16728712

**Comment by mgriese** [comment]
Regarding AT1 cells: in the context of some rare diseases important (pulmonary alveolar lithiasis)(can do that when doing that disease

**Comment by pnrobinson** [comment]
OK, this is https://hpo.jax.org/app/browse/term/HP:0033245
I just corrected the definition from MLB to LB as above.
